samples/readSnippets.js

```diff
@@ -0,0 +1,217 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+'use strict';
+
+async function main(
+  instanceId = 'YOUR_INSTANCE_ID',
+  tableId = 'YOUR_TABLE_ID',
+  readType = 'readRow'
+) {
+  // [START bigtable_reads_row]
+  // [START bigtable_reads_row_partial]
+  // [START bigtable_reads_rows]
+  // [START bigtable_reads_row_range]
+  // [START bigtable_reads_row_ranges]
+  // [START bigtable_reads_prefix]
+  // [START bigtable_reads_filter]
+  const {Bigtable} = require('@google-cloud/bigtable');
+  const bigtable = Bigtable();
+
+  /**
+   * TODO(developer): Uncomment these variables before running the sample.
+   */
+  // const instanceId = 'YOUR_INSTANCE_ID';
+  // const tableId = 'YOUR_TABLE_ID';
+  const instance = bigtable.instance(instanceId);
+  const table = instance.table(tableId);
+  // [END bigtable_reads_row]
+  // [END bigtable_reads_row_partial]
+  // [END bigtable_reads_rows]
+  // [END bigtable_reads_row_range]
+  // [END bigtable_reads_row_ranges]
+  // [END bigtable_reads_prefix]
+  // [END bigtable_reads_filter]
+  switch (readType) {
+    // [START bigtable_reads_row]
+    case 'readRow': {
+      const rowkey = 'phone#4c410523#20190501';
+
```

Comment:
Can we delete the unnecessary newlines here?

Reply:
I like to separate the variable declarations from the API call, but I deleted the line between the call and print so it would be consistent across the samples